process/interface.go

```diff
@@ -151,6 +151,13 @@ type BlockProcessor interface {
 	IsInterfaceNil() bool
 }
 
+// PeerProcessor is the main interface for validators' consensus participation statistics
+type PeerProcessor interface {
+	LoadInitialState(in []*sharding.InitialNode) error
```

Comment:
initial state should be set on constructor, not on separate function.

Reply:
Like also Adi suggested, I left this function for other use cases, but called it in the constructor also